mac-add-route.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+gateway=$(docker-machine ip)
+networkname=$(echo $(basename $(pwd)) | tr -d '[:punct:]')_default
+subnet=$(docker network inspect mesosactor_default -f '{{(index .IPAM.Config 0).Subnet}}')
+sudo route -n add -net ${subnet} ${gateway}
```

Comment:
maybe this will help with the master ui problem, I'll just need to figure out with the master ip is each time, since it will change (unless using host network)

Reply:
This allows access to both the master and the marathon UI, what do you want to see on the agent hosts?